{"name": "test_batch_blackList[args1]", "status": "passed", "steps": [{"name": "执行接口", "status": "passed", "start": 1715834314053, "stop": 1715834314452}, {"name": "响应体断言", "status": "passed", "start": 1715834314452, "stop": 1715834314453}], "attachments": [{"name": "log", "source": "a5150cf8-4aff-4c5e-922a-cdc5933694b4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "args", "value": "{'caseName': '【拉黑用户至黑名单接口】用例2：openid为无效数字', 'method': 'post', 'url': '/cgi-bin/tags/members/batchblacklist', 'data': {'openid_list': 121}, 'excepted': {'statusCode': 200, 'errcode': 40003, 'errmsg': 'invalid openid'}}"}], "start": 1715834314053, "stop": 1715834314457, "uuid": "00387d35-0ec1-47de-817d-08b2a2201e35", "historyId": "321fb532615b92241212aa0cda5fe330", "testCaseId": "49e1c317b0bd0f105d2662a3d2e91abc", "fullName": "testcase.black_manage.test_batch_to_black.Test_userBlack#test_batch_blackList", "labels": [{"name": "feature", "value": "黑名单管理模块"}, {"name": "epic", "value": "微信项目"}, {"name": "story", "value": "拉黑用户至黑名单接口"}, {"name": "parentSuite", "value": "testcase.black_manage"}, {"name": "suite", "value": "test_batch_to_black"}, {"name": "subSuite", "value": "Test_userBlack"}, {"name": "host", "value": "DESKTOP-IG8PJM0"}, {"name": "thread", "value": "4692-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.black_manage.test_batch_to_black"}]}